hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -195,32 +195,32 @@ share/ozone/lib/ozone-insight.jar
 share/ozone/lib/ozone-interface-client.jar
 share/ozone/lib/ozone-interface-storage.jar
 share/ozone/lib/ozone-manager.jar
-share/ozone/lib/ozone-reconcodegen.jar
 share/ozone/lib/ozone-recon.jar
+share/ozone/lib/ozone-reconcodegen.jar
 share/ozone/lib/ozone-s3gateway.jar
 share/ozone/lib/ozone-tools.jar
 share/ozone/lib/perfmark-api.jar
 share/ozone/lib/picocli.jar
+share/ozone/lib/proto-google-common-protos.jar
+share/ozone/lib/protobuf-java-util.jar
 share/ozone/lib/protobuf-java.jar
 share/ozone/lib/protobuf-java.jar
-share/ozone/lib/protobuf-java-util.jar
-share/ozone/lib/proto-google-common-protos.jar
-share/ozone/lib/ratis-client.jar
-share/ozone/lib/ratis-common.jar
-share/ozone/lib/ratis-grpc.jar
-share/ozone/lib/ratis-metrics.jar
-share/ozone/lib/ratis-netty.jar
-share/ozone/lib/ratis-proto.jar
-share/ozone/lib/ratis-server-api.jar
-share/ozone/lib/ratis-server.jar
+share/ozone/lib/ratis-clientdb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-commondb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-grpcdb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-metricsdb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-nettydb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-protodb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-server-apidb58b-SNAPSHOT.jar
+share/ozone/lib/ratis-serverdb58b-SNAPSHOT.jar
```

Comment:
I'm afraid this is a problem and will need to be changed in the next update to ratis.
I found the dependency Check failed because the regular expression was written incorrectly. This bug was triggered when we updated Ratis.
https://github.com/apache/ozone/blob/HDDS-4454/hadoop-ozone/dist/src/main/license/update-jar-report.sh#L33

The ratis jar name we output after executing this command should not contain the Version and COMMIT ID. We can modify the regular expression so that:
ratis-common-2.3.0-94db58b-SNAPSHOT.jar -> ratis-common.jar